pkg/controller/ttlafterfinished/metrics/metrics.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+// TTLAfterFinishedSubsystem - subsystem name used for this controller.
+const TTLAfterFinishedSubsystem = "ttl_after_finished_controller"
+
+var (
+	// timeToDeletionSeconds tracks the time it took to delete the job since it
+	// became eligible for deletion.
+	TimeToDeletionSeconds = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      TTLAfterFinishedSubsystem,
+			Name:           "time_to_deletion_seconds",
+			Help:           "The time it took to delete the job since it became eligible for deletion",
+			StabilityLevel: metrics.ALPHA,
+			// Start with 100ms with the last bucket being [~2h, Inf).
```

Comment:
what sort of time_to_delete_seconds do we typically expect here? are values > 2h uncommon?

Reply:
note that this metric is for the time "since it became eligible", that is, after the TTL.

So, it should happen rather fast (within seconds)

Reply:
yeah, as Aldo mentioned, this is to track how fast the controller is reacting, ideally in the order of seconds, anything over hours doesn't really matter because it is just bad.